### ISSUE TITLE
No search results and 404 pages form needs BS5 update

### ISSUE THREAD

**Issue by chrisblakley** [chrisblakley/Nebula#2168]
The form group is going full width. I'm guessing it is still BS4 syntax maybe?

![Screen Shot 2021-07-12 at 11 54 12 PM](https://user-images.githubusercontent.com/6349678/125387860-90c90b80-e36c-11eb-8675-70bd097b7f24.png)

https://nebula.gearside.com/?s=cheeseburger

**Comment by chrisblakley** [comment]
404 page too:

![Screen Shot 2021-07-12 at 11 57 57 PM](https://user-images.githubusercontent.com/6349678/125388175-092fcc80-e36d-11eb-84da-f0d9fda96067.png)

**Comment by chrisblakley** [comment]
This was a Bootstrap 5 upgrade issue. `inline-form` class no longer exists. It was replaced by using the grid layout classes (row/col) within the form.

Here's a comparison I made: https://codepen.io/chrisblakley/pen/MWmJmyV

![Screen Shot 2021-07-13 at 6 28 50 PM](https://user-images.githubusercontent.com/6349678/125533311-d04e90f1-8735-4b39-bda1-c57028882d77.png)

Fortunately the only update needed was to `search_form()` in Functions.php.
